SonarQube.Bootstrapper/SonarQube.Analysis.xml

```diff
@@ -5,16 +5,16 @@
   is located (if it exists). It is possible to use another properties file by using the /s:filePath.xml flag
   
   The overriding strategy of property values is the following:
-  - A property defined in the command line (/d:propertyName=value) has the top priority and so can override :
-  - A property defined in an MSBuild project file (corresponding to a SonarQube module) which can override :
-  - A property defined in the MSBuild.Analysis.xml configuration file [this file] which can override :
+  - A property defined in an MSBuild project file (*.csproj / *.vbproj) (corresponding to a SonarQube module) which can override:
+  - A property defined in the command line (/d:propertyName=value) has the top priority and so can override:
```

Comment:
Why the order change? It used to be that command line >> msbuild property - has this changed? Also the phrasing suggests the order has not changed - "has the top priority"